src/box/lua/key_def.c

```diff
@@ -62,6 +62,9 @@ luaT_push_key_def(struct lua_State *L, const struct key_def *key_def)
 		lua_pushboolean(L, key_part_is_nullable(part));
 		lua_setfield(L, -2, "is_nullable");
 
+		lua_pushboolean(L, part->exclude_null);
+		lua_setfield(L, -2, "exclude_null");
```

Comment:
We want to keep the [tuple-keydef](https://github.com/tarantool/tuple-keydef) module compatible with the built-in one, so, please, file an issue to its repository, when the change will land to master (or after agreement about the future Lua API). It is nice that you updated the public C API: so we can support it in the external module as well.